### PR TITLE
Improve handler docs

### PR DESCRIPTION
**Pull request by temeddix**
## Changes

This PR clarifies some explanations included in `handler.md` docs file. It also adds an example using `SimpleHandler`.

## Checklist

- [x] An issue to be fixed by this PR is listed above.
- [x] New tests are added to ensure new features are working. End-to-end tests are usually in the `./frb_example/pure_dart` example, more specifically, `rust/src/api.rs` and `dart/lib/main.dart`.
- [x] The code generator is run and the code is formatted (via `just precommit`).
- [x] If this PR adds/changes features, documentations (in the `./book` folder) are updated.
- [x] CI is passing.

## Remark for PR creator

- Justfile is a task runner for the command line interface that allows you to run shell commands from a file named `justfile` in your project directory, You can use Justfile after [installing it](https://github.com/casey/just). Note that commands written in `justfile` of this repository are expected to be run in `bash`, not `cmd` or `powershell`. Running `just ...` commands in `cmd` or `powershell` will produce erorrs as the syntax is not compatible. On Windows, you can use `git bash` if you have Git installed.
- If fzyzcjy does not reply for a few days, maybe he just did not see it, so please ping him.

### CODE REVIEW

File: book/src/feature/handler.md [hunk 1]
```diff
@@ -45,3 +45,19 @@ impl MyExecutor {
...
+
+use flutter_rust_bridge::handler::ReportDartErrorHandler as RDEH;
+use flutter_rust_bridge::handler::SimpleHandler as SH;
+use flutter_rust_bridge::handler::ThreadPoolExecutor as TPE;
+use lazy_static::lazy_static;
+
+lazy_static! {
+    static ref FLUTTER_RUST_BRIDGE_HANDLER: SH<TPE<RDEH>, RDEH> = SH::new(
+        TPE::new(RDEH), RDEH {}
+    );
```
Comment (fzyzcjy) [inline book/src/feature/handler.md:61]: uber-nit: personally prefer not using abbr, especially in the docs, where readability may be quite useful

**Review by fzyzcjy — APPROVED** [review]
LGTM, just one nit :)

File: book/src/feature/handler.md (hunk 1 above)
Comment (temeddix) [reply book/src/feature/handler.md:61]: Okay I'll fix it right away

**Comment by fzyzcjy** [comment]
@all-contributors please add @temeddix for doc

**Pull request merged**